prow/config/external_plugins_config.yaml

```diff
@@ -1121,6 +1121,7 @@ ti-community-label-blocker:
           - qa-release-merge
         trusted_users:
           - ti-chi-bot
+          - lonng
```

Comment:
I think you just need to add Longheng for the sig migrate repositories.

Reply:
You can split the repo config.

Reply:
I see！

Reply:
There are still some problems, each repo can only appear once in repos, so you should copy the complete configuration above before making changes.